app/views/overview/_service-instance-row.html

```diff
@@ -14,13 +14,18 @@
           </div>
         </h3>
         <div class="status-icons">
+          <!--
+            TODO: this should show an info status icon if the apiObject is not yet ready.
+          -->
           <notification-icon ng-if="!row.expanded" alerts="row.notifications"></notification-icon>
+          <span ng-if="!row.isReady">
+            <status-icon status="row.status"></status-icon> {{row.statusReason | sentenceCase}}
```

Comment:
And this should probably be handled in the `<notification-icon>` above like other alerts, correct?